robottelo/host_helpers/cli_factory.py

```diff
@@ -253,7 +253,8 @@ def __getattr__(self, name):
         These are all basic cases where the make method just need some default values.
         For more complex make methods, we define them in methods below.
         """
-        if fields := ENTITY_FIELDS.get(name.replace('make_', '')):
+        fields = ENTITY_FIELDS.get(name.replace('make_', ''))
+        if isinstance(fields, dict):
```

Comment:
`ENTITY_FIELDS.get` returns `None` if e.g`make_abcdefg` doesn't exist
previous solution when  `ENTITY_FIELDS.get` returns `{}`  which is correct scenario formultiple make_entities didn't met the condition if {} -> false, instead of true